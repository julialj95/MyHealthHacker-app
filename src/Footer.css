.footer {
  display: flex;
  flex-direction: row;
  justify-content: space-around;
  position: absolute;
  bottom: 0;
}
