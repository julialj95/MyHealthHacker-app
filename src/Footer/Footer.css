@import url("https://fonts.googleapis.com/css2?family=Lato&family=Nunito&family=Source+Sans+Pro&display=swap");

.footer {
  display: flex;
  flex-direction: row;
  justify-content: space-around;
  color: white;
  font-family: "Nunito";
}

.footer_content {
  font-size: 16px;
}

@media all and (min-width: 600px) {
  .footer_content {
    font-size: 18px;
  }
}
@media all and (min-width: 900px) {
  .footer_content {
    font-size: 21px;
  }
}
