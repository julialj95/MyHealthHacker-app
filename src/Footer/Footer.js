import React from "react";
import "./Footer.css";

function Footer() {
  return (
    <footer className="footer">
      <p className="footer_content">MyHealthHacker</p>
      <p className="footer_content">Â© 2021</p>
    </footer>
  );
}

export default Footer;
