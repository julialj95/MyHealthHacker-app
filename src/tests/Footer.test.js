import React from "react";
import ReactDOM from "react-dom";
import Footer from "../Footer/Footer";
import { MemoryRouter as Router } from "react-router-dom";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");

  ReactDOM.render(
    <Router>
      <Footer />
    </Router>,
    div
  );
  ReactDOM.unmountComponentAtNode(div);
});

it("renders the UI as expected", () => {
  const tree = renderer
    .create(
      <Router>
        <Footer />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
