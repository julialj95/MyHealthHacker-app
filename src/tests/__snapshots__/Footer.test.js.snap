// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<footer
  className="footer"
>
  <p
    className="footer_content"
  >
    MyHealthHacker
  </p>
  <p
    className="footer_content"
  >
    Â© 2021
  </p>
</footer>
`;
